src/codehelp/formatter.vala

```diff
@@ -50,6 +52,9 @@ namespace Vls.Formatter {
             if (stderr_buf != null && stderr_buf.strip ().length > 0) {
                 throw new FormattingError.FORMAT ("%s", stderr_buf);
             } else {
+                foreach (var arg in args)
```

Comment:
I don't think we need to log the arguments. But if you must, don't use `stderr.printf()`. Use something like `GLib.debug()`.

Reply:
I think it would be nice for cases like:

The formatting works for one file, but for the other one uncrustify fails e.g. with an internal error. This would already give a handy command to start troubleshooting

Reply:
We could use [window/showMessage](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#window_showMessage)

Reply:
I just added the command to the error